Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -3451,7 +3451,7 @@ void drawMotionMenu() {
     BACK_ITEM(drawControlMenu);
     MENU_ITEM(ICON_MaxSpeed, MSG_SPEED, onDrawSpeed, drawMaxSpeedMenu);
     MENU_ITEM(ICON_MaxAccelerated, MSG_ACCELERATION, onDrawAcc, drawMaxAccelMenu);
-    #if HAS_CLASSIC_JERK
+    #if ENABLED(CLASSIC_JERK)
       MENU_ITEM(ICON_MaxJerk, MSG_JERK, onDrawJerk, drawMaxJerkMenu);
     #elif HAS_JUNCTION_DEVIATION
       EDIT_ITEM(ICON_JDmm, MSG_JUNCTION_DEVIATION, onDrawPFloat3Menu, setJDmm, &planner.junction_deviation_mm);
```

Comment:
Note for ProUI: After editing `planner.junction_deviation_mm` with `HAS_LINEAR_E_JERK` there should be a call to `planner.recalculate_max_e_jerk` to update the calculated E jerk values since they depend on `planner.junction_deviation_mm`.